code/modules/food_and_drinks/recipes/tablecraft/recipes_pastry.dm

```diff
@@ -23,7 +23,7 @@
 	result = /obj/item/reagent_containers/food/snacks/donut
 	subcategory = CAT_PASTRY
 
-/datum/crafting_recipe/food/donut
+/datum/crafting_recipe/food/donut/semen
```

Comment:
???
what the fuck is this fetish content?
>makes lewd donut and force feeds it to people.
This is just sick

Reply:
don't ask me, i'm just fixing bugs

Reply:
Ill make a poll to remove these and let people vote